### DISCUSSION TITLE
点歌 Causal Representation Learning

### DISCUSSION THREAD

**Discussion by jasperhyp** [mli/paper-reading#175] [General]
尤其是Bengio组近期的大批工作，感觉可以串讲

**Comment by jasperhyp** [comment]
比如这篇：https://papers.nips.cc/paper/2021/file/5989add1703e4b0480f75e2390739f34-Paper.pdf

**Comment by Lkiku** [comment]
顶一顶 我对这个领域也很感兴趣
